src/types/MovieTypes.ts

```diff
@@ -0,0 +1,7 @@
+export interface MovieTypes {
+  title: string,
+  description: string,
+  imdbUrl: string,
```

Comment:
в тайпскрипте свойства описываются через точку с запятой
```suggestion
  title: string;
  description: string;
  imdbUrl: string;
```